docs/framework/deployment/windows/vista.md

```diff
@@ -0,0 +1,23 @@
+# Installing the .NET Framework on Windows Vista
+
+You may need the .NET Framework to run an application on Windows Vista. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+
+## .NET Framework 4.6.1 and later
+
+You cannot install the .NET Framework 4.6.1 or later on Windows Vista. Instead, you must upgrade to a later Windows version to use a later version of the .NET Framework.
```

Comment:
later Windows version to use a later version of ... seems repetitive. how about saying something like 
"You must upgrade to a later version of Windows to use .NET Framework 4.6.1 or later."